templates/install/project.md

```diff
@@ -67,6 +67,12 @@ The keyword `math` at the end of this command adds `mathlib4` to the dependencie
 
 * Go inside the `my_project` folder and type `lake update`, then `lake exe cache get` and then `mkdir MyProject`.
 
+  * Some Windows users have reported an error like this when running `lake exe cache get`:
+    ```
+      curl: (35) schannel: next InitializeSecurityContext failed: Unknown error (0x80092012) - The revocation function was unable to check revocation for the certificate
+    ```
+    If you see this error, you likely have an antivirus program that is unhappy that `lake` downloads the compiled files from mathlib. You can temporarily disable your antivirus program, or tell it to ignore/allow the operation of `lean.exe`. After you do this, run `lake exe cache get!`. The exclamation mark forces `lake` to re-download the cache files it failed to download before.
```

Comment:
I think this is confusing two separate issues:
* Antivirus is messing with all of Lean's IO, slowing it to a crawl
* Antivirus is messing with curl's certificate checking, causing it to complain that the connection has been compromised

The solution is for the first, but the error message is from the second.

Reply:
Better now?